src/assets/i18n/en.json

```diff
@@ -17,21 +17,71 @@
   },
   "debrief": {
     "showMeTellMeQuestions": {
-      "T01": "Brakes",
-      "T02": "Safety factors while loading",
-      "T03": "Safety factors while securing a load",
-      "T04": "Sufficient tread",
-      "T05": "Engine coolant",
-      "T06": "Head restraint",
-      "S01": "Direction indicators",
-      "S02": "Doors secure",
-      "S03": "Horn",
-      "S04": "Parking brake",
-      "S05": "Windscreen",
-      "S06": "Demist windscreen",
-      "S07": "Rear fog lights",
-      "S08": "Dipped to main beam",
-      "S09": "Brake lights"
+      "B": {
+        "T01": "Brakes",
+        "T02": "Safety factors while loading",
+        "T03": "Safety factors while securing a load",
+        "T04": "Sufficient tread",
+        "T05": "Engine coolant",
+        "T06": "Head restraint",
+        "S01": "Direction indicators",
+        "S02": "Doors secure",
+        "S03": "Horn",
+        "S04": "Parking brake",
+        "S05": "Windscreen",
+        "S06": "Demist windscreen",
+        "S07": "Rear fog lights",
+        "S08": "Dipped to main beam",
+        "S09": "Brake lights"
+      },
+      "B+E": {
+        "T01": "Brakes",
+        "T02": "Safety factors while loading",
+        "T03": "Safety factors while securing a load",
+        "T04": "Sufficient tread",
+        "T05": "Engine coolant",
+        "T06": "Head restraint",
+        "S01": "Direction indicators",
+        "S02": "Doors secure",
+        "S03": "Horn",
+        "S04": "Parking brake",
+        "S05": "Windscreen",
+        "S06": "Demist windscreen",
+        "S07": "Rear fog lights",
+        "S08": "Dipped to main beam",
+        "S09": "Brake lights"
+      },
+      "C": {
+        "T1": "Safety factors while loading",
```

Comment:
The removal of leading zeros from the question indicator - can you remind me what was the reason for it? Is it intentional since those TXX/SXX will be sent to RSIS so I'm not sure if that's expected on their end

Reply:
Also, I don't think it will be picked up by the component as it uses the code as the key when selecting the text in the JSON. The code has the leading zero...

Reply:
@rstepnowski so i got the codes form the constants file that exist already in the app (src/shared/constants/show-me-questions/show-me-questions.vocational.constants.ts), i assumed these should've aligned to the values in the translation JSON objects, and therefore copied the codes, if thats not the case should be minor changes

Reply:
Think it's worth checking because I believe with B+E they were needed for the translation mapping etc

Reply:
Yeah, we've asked that question to RSIS guys - they don't mind atm but consistency sake we want to remove the leading zeros to align all of the categories with cat B.